puzzlejs/puzzle.js

```diff
@@ -2,7 +2,8 @@
  * This code is licensed under the MIT License.
  * https://github.com/tabascq/PuzzleJS
  */
-var puzzleJsFolderPath = document.currentScript.src.replace("puzzle.js", "");
+//var puzzleJsFolderPath = document.currentScript.src.replace("puzzle.js", "");
+var puzzleJsFolderPath = document.currentScript.src.split("puzzle.js")[0];
```

Comment:
I use the trick of doing "?v#" after a filename in order to force the browser to reload (when that # is changed) instead of using a stale, cached version. With the original code, that left "?v2" at the beginning of the path, which got pre-pended to the file it was trying to load (svg line stuff). My current line:
<script src="puzzlejs/puzzle.js?v4" language="JavaScript" type="text/javascript"></script>

Reply:
That makes sense, thanks for the explanation!